bonsai/Data/Entity/Transaction.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Transaction.swift
+//  bonsai
+//
+//  Created by Vladimir Korolev on 06.12.2021.
+//
+
+import Foundation
+
+struct Transaction: Identifiable {
+    let id: UUID
+
```

Comment:
no need empty space

Reply:
Empty space here separates Identifiable conformance from actual entity

I think we should not discuss code-style in the PR's but instead focus on logic, bugs etc

Reply:
Agree here. I suppose empty spaces aren't really important considering that we're building MVP